src/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { FC, ReactNode, useContext, useEffect, useReducer } from "react"
+import { useDidMount } from "Utils/Hooks/useDidMount"
+import { filterReducer } from "./artworkFilterReducer"
+import { hasFilters } from "./Utils/hasFilters"
+import { isDefaultFilter } from "./Utils/isDefaultFilter"
+import { rangeToTuple } from "./Utils/rangeToTuple"
+import { urlFragmentFromState } from "./Utils/urlFragmentFromState"
+
+export interface FilterContextValues {
+  filters: Filters
+  hasFilters: boolean
+  isDefaultValue: (name: string) => boolean
+  rangeToTuple: (name: string) => [number, number]
+  resetFilters: () => void
+  setFilter: (name: string, value: any) => void
+  unsetFilter: (name: string) => void
+}
+
+interface FilterContextProps extends Filters {
+  children: ReactNode
+}
+
+// TODO - consolidate these initial values....
```

Comment:
Note that `TODO -` formatting like this isn't recognized by editor highlighting; need to write `TODO:`